scripts/rhel_validation/take_kernel_version.ps1

```diff
@@ -0,0 +1,166 @@
+################################################################################
+#
+# Linux on Hyper-V and Azure Test Code, ver. 1.0.0
+# Copyright (c) Microsoft Corporation
+#
+# All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the ""License"");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION
+# ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR
+# PURPOSE, MERCHANTABLITY OR NON-INFRINGEMENT.
+#
+# See the Apache Version 2.0 License for specific language governing
+# permissions and limitations under the License.
+#
+################################################################################
+
+################################################################################
+#
+#	Description
+#
+#	This script will find out if new kernel exists for each version of Redhat
+# from RHEL6.0 to RHEL7.4.
+#	This script imports the login cookies for downloading the html to get 
+# the entire list of kernels. For each version of Redhat it creates a list with 
+# kernels associated and stores these informations in a hash table.
+#	Each version of Redhat has a latest kernel stored in a file which is compared 
+# with the last added kernel in hash table.
+#
+################################################################################
+################################################################################
+#
+# Main script body
+#
+################################################################################
+
+param (
+    [String] $WorkDir,
+    [String] $LatestVersionFile,
+    [String] $KernelVersionsPath,
+    [String] $UtilsDir
+)
+
+if (!(Test-Path $WorkDir)) {
```

Comment:
this is a no method script, create the corresponding method plus main one and call main at the end